shared/macros/10-bash.jinja

```diff
@@ -847,6 +847,45 @@ fi
 {{%- endmacro -%}}
 
 
+{{#
+    Enable pam_unix.so PAM module by using pam-auth-update.
+    This option is only recommended when pam-auth-update tool is available for the system.
+#}}
+{{%- macro bash_pam_unix_enable() -%}}
+conf_name=cac_unix
+if [ ! -f /usr/share/pam-configs/"$conf_name" ]; then
+    cat << EOF > /usr/share/pam-configs/"$conf_name"
+Name: Unix authentication
+Default: yes
+Priority: 257
+Conflicts: unix
+Auth-Type: Primary
+Auth:
+        [success=end default=ignore]    pam_unix.so try_first_pass
```

Comment:
This line should contain nullok, same as in the default unix file.
Rule `no_empty_passwords` or similar should take care of removing the nullok.